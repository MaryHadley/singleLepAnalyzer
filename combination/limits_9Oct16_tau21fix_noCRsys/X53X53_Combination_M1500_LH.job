universe = vanilla
Executable = X53X53_Combination_M1500_LH.sh
Should_Transfer_Files = YES
WhenToTransferOutput = ON_EXIT
Notification = Error
request_memory = 3072
Output = X53X53_Combination_M1500_LH.out
Error = X53X53_Combination_M1500_LH.err
Log = X53X53_Combination_M1500_LH.log
Queue 1