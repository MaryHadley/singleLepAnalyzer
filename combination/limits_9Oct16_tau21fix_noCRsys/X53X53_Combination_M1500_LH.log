000 (1297369.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297369.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.233:40949>
...
006 (1297369.000.000) 10/10 19:27:26 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2268  -  ResidentSetSize of job (KB)
...
006 (1297369.000.000) 10/10 19:32:26 Image size of job updated: 282732
	161  -  MemoryUsage of job (MB)
	164052  -  ResidentSetSize of job (KB)
...
006 (1297369.000.000) 10/10 20:12:31 Image size of job updated: 283800
	162  -  MemoryUsage of job (MB)
	164924  -  ResidentSetSize of job (KB)
...
005 (1297369.000.000) 10/10 20:15:51 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:48:18, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:48:18, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1631  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1631  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1095        1   3824816
	   Memory (MB)          :      162     3072      3072
...
