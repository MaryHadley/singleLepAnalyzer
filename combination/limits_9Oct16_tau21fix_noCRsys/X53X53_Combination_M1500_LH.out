/var/lib/condor/execute/dir_9366
Total number of likelihood evaluations: 55000101
Total number of likelihood evaluations: 5500101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_LH/quant-toys:0-sig-a33d8d1c2c.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_LH/quant-data-sig-1973bd7447.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1500_LH
