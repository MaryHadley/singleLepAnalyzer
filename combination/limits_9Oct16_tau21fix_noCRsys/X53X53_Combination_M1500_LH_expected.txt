# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.077753   0.039354    0.15971  0.053905    0.11262
